app/main.py

```diff
@@ -1 +1,6 @@
-# write your code here
+def copy_file(command: str) -> None:
+    ls_command = command.split(" ")
+    if ls_command[1] != ls_command[2] and ls_command[0] == "cp":
+        with (open(ls_command[1], "r") as source,
+              open(ls_command[2], "w") as new_file):
```

Comment:
it's better to unpack values into the variables, like: `var, var, var = parts`. Then use these variables instead of indexes, it will increase readability